README.md

```diff
@@ -114,25 +112,25 @@ Command                                    | Task
 
 To create functional test pages from a schema use the following command.
 
-`generate_pages.py <schema.json> <page_directory>` 
+`generate_pages.py <schema.json> <page_directory>`
 
 This can combined with `-spec_file=<spec_file_path>` to provide a populated template for the functional tests spec file.
 and/or `-require_path='../..` To provide a relative path from a page file to the directory containing the base/parent page classes. Defaults to ".."
 
-For example: 
-`./generate_pages.py ../../data/en/test_navigation.json ./pages/surveys/navigation --spec_file=./spec/navigation.spec.js` 
+For example:
+`./generate_pages.py ../../data/en/test_navigation.json ./pages/surveys/navigation --spec_file=./spec/navigation.spec.js`
 
 The functional tests can be executed with a couple of options.
 
-`--env=preprod` Will run the test against preprod.
-`--spec=mci` Will run a single test spec (`mci.spec.js`) instead of the entire suite.
-`--suite=core` Will run a suite of tests (core) instead of the entire suite.
+`--path=tests/functional/spec/components/` Will run all of the tests under this directory.
+
+These options can be combined with `test_functional` eg:
 
-These options can be combined with `test_functional` or `test_functional_sauce`, eg:
+`yarn test_functional -- --path tests/functional/spec/components/` Will run all of the component tests test.
```

Comment:
I don’t think you need the extra `--` in the yarn command - `From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.`